README.md

```diff
@@ -1 +1,24 @@
-# desta451616-hotmail.com.github.io
\ No newline at end of file
+# Project Name:portfolio
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
```

Comment:
- [ OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd repo name` etc